dojo/test/queries.py

```diff
@@ -5,21 +5,25 @@
 from dojo.authorization.authorization import get_roles_for_permission, user_has_global_permission
 
 
-def get_authorized_tests(permission, product=None):
+def get_authorized_tests(permission, product=None, testID=None):
```

Comment:
what is this param for? please rename it to a pythonic way, i.e. test_id

Reply:
I don't like that parameter, the queries shall be as universal as possible. And I can't see in this PR where it shall be used.